packages/devtools_app/lib/src/screens/profiler/cpu_profile_columns.dart

```diff
@@ -5,84 +5,40 @@
 import 'package:flutter/material.dart';
 
 import '../../shared/globals.dart';
-import '../../shared/primitives/utils.dart';
 import '../../shared/routing.dart';
 import '../../shared/table/table.dart';
 import '../../shared/table/table_data.dart';
-import '../../shared/utils.dart';
 import '../debugger/codeview_controller.dart';
 import '../debugger/debugger_screen.dart';
 import '../vm_developer/vm_developer_common_widgets.dart';
 import 'cpu_profile_model.dart';
 
-const timeColumnWidthPx = 180.0;
-
-// TODO(kenz): clean up to use TimeAndPercentageColumn.
-class SelfTimeColumn extends ColumnData<CpuStackFrame> {
-  SelfTimeColumn({String? titleTooltip})
-      : super(
-          'Self Time',
+class SelfTimeColumn extends TimeAndPercentageColumn<CpuStackFrame> {
+  SelfTimeColumn({
+    String? titleTooltip,
+    InlineSpan? Function(CpuStackFrame, BuildContext)? dataTooltipProvider,
```

Comment:
Nit: make a `typedef` for this function type if we're going to use it in more than one place.